Content.Server/Motd/MOTDSystem.cs

```diff
@@ -0,0 +1,99 @@
+using Content.Server.Chat.Managers;
+using Content.Server.GameTicking;
+using Content.Shared.CCVar;
+using Content.Shared.Chat;
+using Robust.Server.Player;
+using Robust.Shared.Console;
+using Robust.Shared.Configuration;
+
+namespace Content.Server.Motd;
+
+public sealed class MOTDSystem : EntitySystem
+{
+    #region Dependencies
```

Comment:
i dont think a region is necessary with so few dependencies

Reply:
Done